### PR TITLE
Create notification service

### PR DESCRIPTION
**Pull request by simonbu11**

### CODE REVIEW

**Comment by simonbu11** [comment]
Service side of processing notification requests. Client side coming in different repo

**Comment by simonbu11** [comment]
Client [PR now available](https://github.com/DFE-Digital/login.dfe.notifications.client/pull/1)

File: src/config/index.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+const getSettingsObject = (settings) => {
+  try {
+    return JSON.parse(settings);
+  } catch (e) {
+    return null;
+  }
+};
+
+const getSettingsFromFile = (settingsPath) => {
+  if (fs.existsSync(settingsPath)) {
```
Comment (dashton82) [inline src/config/index.js]: should this be statSync().isFile() as it will return successfully for that if its just the path?

**Review by dashton82 — APPROVED** [review]

File: src/config/index.js (hunk 1 above)
Comment (simonbu11) [reply src/config/index.js]: Good point. though docs actually say not to use stat of existence checks before fileRead. So I will just remove it

**Pull request merged**